src/rt/sched/behaviourcore.h

```diff
@@ -759,6 +793,9 @@ namespace verona::rt
         Slot* last_slot;
         size_t transfer_count;
         bool had_no_predecessor;
+        // The last two are only use for reads only chains
+        size_t ref_count;
+        size_t ex_count;
```

Comment:
Could this be merged with the `ec` array?  Or the `ec` array be removed and just use this?